chain.go

```diff
@@ -46,19 +46,23 @@ func NewChain() (chain *Chain) {
 		Hmap:     make(map[string]int),
 		Emap:     make(map[string]int),
 	}
-	chain = &c
+	holo.chain = &c
 	return
 }
 
 // NewChainFromFile creates a chain from a file, loading any data there,
 // and setting it to be persisted to. If no file exists it will be created.
-func NewChainFromFile(h HashSpec, path string) (c *Chain, err error) {
+func (holo *Holochain) NewChainFromFile() (err error) {
```

Comment:
I think maybe this name should change. The other one (NewChain) really just creates the chain (in memory like most New* methods. This one actually creates the storage when it doesn't exist, right? InitChain() seems right to me.